test/ibmq/websocket/test_websocket_integration.py

```diff
@@ -217,3 +218,18 @@ def test_websocket_proxy_invalid_port(self):
                 job.wait_for_final_state()
 
         self.assertIn("retrying using HTTP", ','.join(log_cm.output))
+
+    def _run_job(self, backend: Backend = None, max_retries: int = 10,
+                 qc: Union[QuantumCircuit, List[QuantumCircuit]] = None, **kwargs) -> IBMQJob:
```

Comment:
Since you have the `_run_job` duplicated in 3 different test modules, perhaps it'd be better to put it in `utils` instead?

Reply:
Updated! Moved to `test/utils.py`